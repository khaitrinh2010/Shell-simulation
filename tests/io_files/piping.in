echo "My name is Khai" | cat
cat students.csv | sort -r | cut -d, -f1
