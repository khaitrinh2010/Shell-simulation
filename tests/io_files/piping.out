>> My name is Khai
>> MATH1064
INFO3001
INFO2112
INFO2005
INFO2001
INFO1112
INFO1001
ELEC1601
Code
>> 
